services/src/WebAPI/Controllers/BlockUser/BlockUserQueryController.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Application.BlockUser.Queries.GetAllBlockedUsers;
+using Application.Models;
+using Application.Models.Pagination;
+using Domain.Common.Pagination.Response;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using WebAPI.Common.Authorization;
+
+namespace WebAPI.Controllers.BlockUser;
+
+[ApiController]
+[AuthorizeUser]
+[Route("api/v1/blocked-users")]
+public class BlockUserQueryController : ControllerBase
```

Comment:
1. Could be sealed
2. There should be also endpoint to check if currently logged user is blocked